CHANGES.rst

```diff
@@ -1,16 +1,20 @@
 Changes
 =======
 
-Version 0.7.3 (UNRELEASED)
+Version 0.7.3 (2021-03-17)
 --------------------------
 
+- Users:
+    - Fixes problem with jobs stuck in `running` state when there are network issues, improving the overall platform resilience.
```

Comment:
- Fixes problem with failed jobs being reported as still running in case of network problems.
- Changes job status consumer communications to improve overall platform resilience.